.github/workflows/test.yml

```diff
@@ -182,15 +182,19 @@ jobs:
         run: |
           which pg_isready || sudo apt-get update && sudo apt-get install -y postgresql-client
           make DOCKER_ARGS='-d' PG_OPTS='-c shared_buffers=256MB -c max_connections=200000' -C testing/dbtest/docker database-up
-          until pg_isready -h 127.0.0.1; do docker container inspect boundary-sql-tests &> /dev/null || exit 255; sleep 1; done
+          until pg_isready -h 127.0.0.1; do docker container inspect boundary-sql-tests &> /dev/null || exit -1; sleep 1; done
```

Comment:
I think this doesn't need to change.

Reply:
Ah sorry, I think this was a result of a merge conflict, I'll fix that